docker/install/ubuntu2004_install_python_package.sh

```diff
@@ -43,4 +43,5 @@ pip3 install --upgrade \
     junitparser==2.4.2 \
     six \
     tornado \
-    pytest-lazy-fixture
+    pytest-lazy-fixture \
+    git+https://github.com/jax-ml/ml_dtypes.git@v0.2.0
```

Comment:
I think we should favour the usage of this package from Pypi, like other platforms. I checked and the source package is there: https://pypi.org/project/ml-dtypes/0.2.0/#files

Can we just move to a plain `ml_dtypes` type of dependency?

Reply:
I was using ml_dtypes from PyPI at first but unfortunately, there is no i686 prebuilt so it will fallout to compile from source (`.tar.gz` file), and it doesn't work (see this flow: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-docker/detail/PR-15226/2/pipeline).